src/main/java/data/UserData.java

```diff
@@ -0,0 +1,19 @@
+package data;
+
+import base.BasePage;
+import configurationManager.BaseConfiguration;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import pages.HomePage;
+import pages.LogInPage;
+
+public class UserData {
+
+    public static String salesUserEmail = "vladyslav.holi+1ver@ctdev.io";
+    public static String salesUserPassword ="Testtest1";
+
+    public static String adminUserEmail = "vladyslav.holiver@curious-moose-50xavl.com";
+    public static String adminUserPassword = "Test_123";
```

Comment:
This variables are constant, it shouldn't be changed during running a test, that's why 'final' should be added (like public static final). All static variables should be capital (e.g. SALES_USER_EMAIL).

Reply:
done